include/SDL3/SDL_test_log.h

```diff
@@ -51,6 +51,16 @@ extern "C" {
  */
 void SDLTest_Log(SDL_PRINTF_FORMAT_STRING const char *fmt, ...) SDL_PRINTF_VARARG_FUNC(1);
 
+/**
+ * Prints given prefix and raw data with a timestamp in the TEST category and INFO priority.
+ * Non-printible characters in the raw data are substituted by printible alternatives.
+ *
+ * \param prefix Prefix message.
+ * \param raw_data Raw data.
+ * \param len Number of bytes in raw_data.
+ */
+void SDLCALL SDLTest_LogRawData(const char *prefix, const Uint8 *raw_data, size_t len);
```

Comment:
Has anyone a suggestion for a better api?
Python has `%r`, but that is not really posix behavior, or supported by `SDL_PRINTF_FORMAT_STRING`.